typed-racket-lib/typed-racket/rep/type-rep.rkt

```diff
@@ -579,6 +579,17 @@
   [#:frees (λ (f) (make-invariant (f value)))]
   [#:key 'continuation-mark-key])
 
+;; Newtype
+;; comes from define-new-subtype
+;; nm: a symbol representing the name of the Newtype
+;; id: a symbol created with gensym
+;; ty: a type for the representation
+(def-type Newtype ([nm symbol?] [id symbol?] [ty Type/c])
```

Comment:
What though? I can't exactly call it `Subtype` or `New-Subtype` to match `define-new-subtype`, so I'm not sure what to do with this.